Dockerfile

```diff
@@ -245,9 +245,21 @@ USER buildbot
 RUN /bin/bash -c 'phpbrew init && source ~/.phpbrew/bashrc && phpbrew install 5.6 +default && \
     phpbrew app get composer'
 
+################################################################################
+#
+# Emacs Cask
+#
+################################################################################
 USER root
+RUN add-apt-repository ppa:kelleyk/emacs
```

Comment:
It looks like orgmode should work just fine with v24.4:

http://orgmode.org/Changes.html

> From now on, Org expects at least Emacs 24.3, although Emacs 24.4 or above is suggested.

I'd rather use the distro version if the library works the same. We don't need all the emacs features in this case.